### ISSUE TITLE
Clickable links in console

### ISSUE THREAD

**Issue by kningyi** [SitecorePowerShell/Console#1292]
Hi,

I was wondering that since Sitecore IDs are clickable in the console (leading to the Sitecore item in Content Editor), is there a way to make clickable urls?

e.g. I would like to be able to click on a url from the console to open in a new tab, or to download from that url.

Also, I notice the ID links only lead to the the item in the shell context language, even if there's no version for that language. 
Is there a way to open to a different language instead? Like say, open to the current language of the ISE window.

**Comment by michaellwest** [comment]
Do you have a sample script or command you run that produces data with a url?

**Comment by kningyi** [comment]
Like these yellow clickable links:

![Image](https://user-images.githubusercontent.com/13121503/202749114-1366c822-682c-4e93-9328-96487566993b.png)

**Comment by michaellwest** [comment]
It's more common to write reports where the column values contain the hyperlinks. In the case of Urls, those are generally generated based on Sitecore context.

This appears to work OOTB.

```
[pscustomobject]@{"Url"="https://doc.sitecorepowershell.com/"}
```

**Comment by kningyi** [comment]
Hmm.. so is it just my system that's not working? :(

![image](https://user-images.githubusercontent.com/13121503/202981967-38c2c175-f1e6-4139-a5e7-7da2b4fa70e1.png)

**Comment by michaellwest** [comment]
I see, the console and ISE have different formatting behavior.

![image](https://user-images.githubusercontent.com/933163/203154137-16c03f7c-795c-480b-b947-a89a74492027.png)

**Comment by kningyi** [comment]
Nice.

While we are at it, is it possible to use the current ISE language when clicking on the ID link?

![image](https://user-images.githubusercontent.com/13121503/205008638-c389ad2c-e25b-4379-ba5f-4f08c5dc57f8.png)

e.g. `item:load(id={B82B0E53-3EB6-4AA6-9E39-60BE22426E02},language=en-GB)`

**Comment by AdamNaj** [comment]
That's not stale. @michaellwest actually implemented this and it's released in 7.0.
